quickjs.c

```diff
@@ -62,7 +62,7 @@
 #define NO_TM_GMTOFF
 #endif
 
-#if !defined(EMSCRIPTEN) && !defined(__wasi__)
+#if !defined(EMSCRIPTEN) && !defined(__wasi__) && defined(__STDC_NO_ATOMICS__) && __STDC_NO_ATOMICS__
```

Comment:
Shouldn't this be negated? We want the check to succeed if __STDC_NO_ATOMICS__ is not defined, right?

Reply:
Hmmm, yea you are totally right, I've messed it up XD

Reply:
Idk how it passed the ci jobs for r2, magic XDD